### PR TITLE
Update label-property-configuration.md

### PR DESCRIPTION
**Pull request by mistyn8**
Add sample usage for the mediaItemResolver

### CODE REVIEW

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md [hunk 1]
```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
 | [umbCmsTitleCase](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsTitleCase) | Converts a string to title case                         | String                      |                                                              |
 | [umbCmsJoinArray](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsJoinArray) | Joins an array into one string                          | Array (of string or object) | separator: string used to join values together, e.g. "`, `"<br />prop (optional): string key indicating which property to join when used on an array of objects |  
 
+### Example usage
+
+#### mediaItemResolver
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:30]: I'm afraid H4 (####) is not going to be visible on the GitBook platform.

How about combining the two headings into something like this:

> ### Example: `mediaItemResolver`

What do you think?

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md [hunk 2]
```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
...
+Assuming a media picker of type Umbraco.MediaPicker3 with multiple images, and taking the first image
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:31]: The line below will render out the name of the image, right?

```suggestion
This is how you could use the filter for a Media Picker of type `Umbraco.MediaPicker3` with multiple images:
```

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md [hunk 3]
```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
...
+`{{(myPropertyAlias[0].mediaKey | mediaItemResolver).name}}`
+
+**mediaKey** is a Guid like *c7a4526c-6b32-4665-a047-5b3e7256d973*
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:34]:
```suggestion
The `mediaKey` is a Guid like `c7a4526c-6b32-4665-a047-5b3e7256d973`.
```

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md [hunk 4]
```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
...
+
+For the Media Name the same could be achieved with ncNodeName
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:36]:
```suggestion
For the Media Name the same could be achieved using `ncNodeName` like shown below:
```

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md [hunk 5]
```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
...
+`{{myPropertyAlias[0].mediaKey ? ('umb://media/'+myPropertyAlias[0].mediaKey.split("-").join("") | ncNodeName) : 'No Image' }}`
+
+**ncNodeName** expectes a udi like *umb://media/c7a4526c6b324665a0475b3e7256d973*
```
Comment (sofietoft) [inline 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:39]:
```suggestion
The `ncNodeName` value expects a UDI like `umb://media/c7a4526c6b324665a0475b3e7256d973`.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for the PR @mistyn8 ! 💪 
Great to have an example of this!

I've made some suggestion, to make it a little more consistent with the rest of the article.
Let me know what you think!

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md (hunk 2 above)
Comment (mistyn8) [reply 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:31]: yep would render out the media node name.

File: 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md (hunk 1 above)
Comment (mistyn8) [reply 11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md:30]: yep no issue with that

**Review by sofietoft — APPROVED** [review]
Thanks for applying the suggestions @mistyn8 ! 💪 

I made some minor changes to the code snippets - looks better with codeblocks on GitBook for this, I think. And it would also be quicker to copy.

I'll merge this as soon as it's done verifying 💪 
Thanks again for the PR!

**Comment by mistyn8** [comment]
Onething I don't particularly like about code snippets is the wrapping (absence of it)
image

**Comment by sofietoft** [comment]
Yeah, that's a good point!
There is an option to add wrapping to the code blocks, making it look like this: 

image

It does add the line break in a funky place though, but at least you can see the entire thing and still copy it.
What do you think?

**Pull request merged**